exasol/nb_connector/cli/processing/processing.py

```diff
@@ -60,20 +64,55 @@ def save(
             continue
         if secret := option.get_secret(interactive=bool(value)):
             scs.save(option.scs_key, secret)
-    return 0
 
 
-def show_scs_content(scs_file: Path) -> int:
+def verify_connection(scs: Secrets) -> None:
+    """
+    Verify if successful connection to the configured backend is possible.
+    Raise an ScsCliError otherwise.
+    """
+    if BackendSelector(scs).use_itde:
+        # Question: Is it OK, to let bring_itde_up modify the SCS content, here?
+        False and bring_itde_up(scs)
+        report.warning(f"Bring up ITDE currently disabled")
+        return
+    try:
+        open_pyexasol_connection(scs).execute("SELECT 1 FROM DUAL").fetchone()
+    except Exception as ex:
+        raise ScsCliError(f"Failed to connect to the configured database {ex}")
```

Comment:
Not sure, if only showing the exception message is helpful, maybe we should format the whole exception with stack trace